tools/dynamic/base_schema.py

```
@@ -0,0 +1,158 @@
+        return []
+    if not base_path.is_file():
+        return []
+
+    order = base_view_order(base_path, view)
+    if not order:
+        return []
+
+    language = page_language(page_path)
+    try:
+        localized_labels = labels_for_language(base_path, language) if language else {}
+    except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)